.github/workflows/ci.yml

```diff
@@ -117,10 +118,9 @@ jobs:
       - name: Checkout Pex
         uses: actions/checkout@v3
       - name: Setup Python ${{ join(matrix.python-version, '.') }}
-        uses: gabrielfalcao/pyenv-action@v14
```

Comment:
As noted in https://github.com/pantsbuild/actions/pull/24 - the caching for gabrielfalcao/pyenv-action is manual and I could not get it to work; thus the switch here.